frontend/src/metabase/dashboard/actions/parameters.js

```diff
@@ -334,13 +335,18 @@ export const toggleAutoApplyFilters = createThunkAction(
   isEnabled => (dispatch, getState) => {
     const dashboardId = getDashboardId(getState());
 
-    dispatch(
-      setDashboardAttributes({
-        id: dashboardId,
-        attributes: { auto_apply_filters: isEnabled },
-      }),
-    );
-    dispatch(saveDashboardAndCards(true));
+    if (dashboardId) {
```

Comment:
This is just to satisfy TypeScript (if we migrate this file to TypeScript) because `dashboardId` is nullable. And I type the parameter in `trackAutoApplyFiltersDisabled` to be non-nullable `boolean`.